### ISSUE TITLE
Bot crashed

### ISSUE THREAD

**Issue by VashSan** [VashSan/wishmaster#9]
Probably on incoming host (must be confiremd still though), the bot crashed with following stack trace:
```
TypeError: this.tags.getAvailableTags(...) is not a function or its return value is not iterable
    at TagReader.assignValues (c:\Users\Vash\Documents\Wishmaster\src\shared\TagReader.ts:58:41)
    at new TagReader (c:\Users\Vash\Documents\Wishmaster\src\shared\TagReader.ts:45:14)
    at Harvest.updateUser (c:\Users\Vash\Documents\Wishmaster\src\Features\Harvest.ts:85:18)
    at Harvest.act (c:\Users\Vash\Documents\Wishmaster\src\Features\Harvest.ts:26:14)
    at MessageProcessor.invokePlugins (c:\Users\Vash\Documents\Wishmaster\src\MessageProcessor.ts:167:19)
    at MessageProcessor.process (c:\Users\Vash\Documents\Wishmaster\src\MessageProcessor.ts:133:14)
    at MessageProcessor.client.onMessage (c:\Users\Vash\Documents\Wishmaster\src\MessageProcessor.ts:59:18)
    at listenerList.forEach.listener (c:\Users\Vash\Documents\Wishmaster\src\ChatClient.ts:180:13)
    at Array.forEach (<anonymous>)
    at TwitchChatClient.invokeAll1 (c:\Users\Vash\Documents\Wishmaster\src\ChatClient.ts:179:22)
```

commit this happened with:
https://github.com/VashSan/wishmaster/commit/8f3469565e1aafc3ddcb9a7dc76579107c37eae2

**Comment by VashSan** [comment]
It is an incoming host. This is a regular incoming message, but it contains now tags at all. 
![grafik](https://user-images.githubusercontent.com/2479263/73497176-75f21980-43ba-11ea-9ea6-c6e6a65599ff.png)

Also the message is created with the regular message handler, I think the tags parameter is wrong here as well, it contains the message from the IRC client. I guess this happened because we are used to process raw messages mostly, but in this case the regular message handler is invoked as well.

ChatClient.ts:185 getMessageHandler()
```typescript
let message: ITaggedMessage = {
                from: from,
                channel: to,
                text: text,
                tags: tags
            };
            this.invokeAll1(this.messageListener, message);
```

There are no tags, so we should not create a tagged message in the first place

**Comment by VashSan** [comment]
Should be fixed by 97657f8795a548f97161b3a98a3c88b4cc8ea12f
